### ISSUE TITLE
A11Y: DataTable - Unable to tab through columns

### ISSUE THREAD

**Issue by rob-richards** [carbon-design-system/carbon#2544]
## Detailed description

> Current Behavior

You press tab into a row and start on column 2: the name, then click tab again and are in the overflow menu, passing right over all other columns. Arrow keys do not work at all.

> Desired Behavior

You should be able to navigate through the columns with the tab or arrow keys.

This issue can be seen on [storybook](http://react.carbondesignsystem.com/?selectedKind=DataTable&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel).

**Comment by dakahn** [comment]
Columns now have tab functionality in latest 👍
